<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>klasaglowna</class>
 <widget class="QMainWindow" name="klasaglowna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>klasaglowna</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="okno3d" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>rotacja</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_rot_x">
          <property name="maximum">
           <number>5760</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>2880</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_rot_y">
          <property name="maximum">
           <number>5760</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>2880</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_rot_z">
          <property name="maximum">
           <number>5760</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>2880</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>przesuniecie</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_odl_x">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_odl_y">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_odl_z">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>-10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>kolor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_kol_r">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_kol_g">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_kol_b">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>6</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_2"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_5"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_4"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_6"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLCDNumber" name="lcdNumber_8"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_7"/>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_9"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>okno3d</class>
   <extends>QOpenGLWidget</extends>
   <header>okno3d.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_rot_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_rot_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_rot_z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_odl_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_odl_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_4</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_odl_z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_6</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_kol_r</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_8</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_kol_g</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_7</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_kol_b</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_9</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
